bindings/huawei/obs/obs_test.go

```diff
@@ -265,7 +265,7 @@ func TestCreateOperation(t *testing.T) {
 			Data: []byte(`"Hello OBS"`),
 		}
 
-		_, err := mo.create(req)
+		_, err := mo.create(context.TODO(), req)
```

Comment:
Please use `context.Background()` in tests

Reply:
![image](https://user-images.githubusercontent.com/20457624/176377616-10817292-2725-4c14-85d1-fd780140f81a.png)
How should we unify?  `TODO()` or `Background()`

Reply:
Background is better. TODO is equivalent but it's meant as "placeholder". If you know you want a background context, use Bakcground.

Don't worry about the other files.... we'll get to that one day :)